.secrets.baseline

```diff
@@ -1,9 +1,9 @@
 {
   "exclude": {
-    "files": "package-lock.json|^.secrets.baseline$",
+    "files": "package-lock.json|go.sum|^.secrets.baseline$",
```

Comment:
why would we have a go.sum file in the python sdk project?

Reply:
My bad, ran the command from the history without modification.